distribution/ddf-common/src/main/resources/etc/org.ops4j.pax.logging.cfg

```diff
@@ -180,9 +180,8 @@ log4j2.appender.contentDirectoryMonitor.name = cdm
 log4j2.appender.contentDirectoryMonitor.fileName = ${karaf.log}/cdm.log
 log4j2.appender.contentDirectoryMonitor.filePattern = ${karaf.log}/cdm.log-%d{yyyy-MM-dd-HH}-%i.log.gz
 log4j2.appender.contentDirectoryMonitor.append = true
-log4j2.appender.contentDirectoryMonitor.ignoreExceptions = false
 log4j2.appender.contentDirectoryMonitor.layout.type = PatternLayout
-log4j2.appender.contentDirectoryMonitor.layout.pattern = [%-5p] %d{ISO8601} | %-16.16t | %-15.20c{1} |  %m%n
+log4j2.appender.contentDirectoryMonitor.layout.pattern = ${log4j2.pattern}
```

Comment:
@SmithJosh, I'm starting a thread to reply to https://github.com/codice/ddf/pull/6344#pullrequestreview-495120963

The xml-style didn't work in the itests. Also, we decided to keep maintaining both here (rather than just use the cfg-style in the itests) because there was some karaf console bug with the xml-style config, which make the cfg-style config preferable for devs who don't need to harden. Lastly, there was a bug with the failover appender that made it so that we needed to use the xml-style if you wanted failover logging, which is part of hardening. These issues might have been fixed since I last worked with these files, but iirc, those were the motivations for this decision.

https://github.com/codice/ddf/blob/bfbd3b4555386b808f4a19e286e2a4d0b30df1bb/distribution/ddf-common/src/main/resources/etc/org.ops4j.pax.logging.cfg#L20-L24

Reply:
Thanks for the explanation. Always wondered about that. The log4j config seems much better all around: well known format, easier to read (imo), and the lines don't get randomly sorted when you go to edit it. But that makes sense